source/slang/slang-check-decl.cpp

```diff
@@ -3702,6 +3749,42 @@ namespace Slang
         checkCallableDeclCommon(decl);
     }
 
+    void SemanticsDeclHeaderVisitor::visitSetterDecl(SetterDecl* decl)
+    {
+        // Make sure to invoke the common checking logic for all accessors.
+        visitAccessorDecl(decl);
+
+        auto params = decl->getParameters();
+        ParamDecl* param = nullptr;
+        if( params.getCount() >= 1 )
+        {
+            param = params.getFirst();
+            if( params.getCount() > 1 )
+            {
+                // TODO: diagnose it!
```

Comment:
Is it okay to just continue here?

Reply:
I probably need to actually implement that `TODO` comment and diagnose the problem.